src/Transformer/DateTimeIso8601Transformer.php

```diff
@@ -14,15 +14,16 @@ class DateTimeIso8601Transformer extends AbstractTransformer
     public function transform(ResourceInterface $resource)
     {
         $value = $this->getPropertyValueFromResource($resource);
+        if (empty($value)) {
```

Comment:
I think so that carbon parser doesn't create now(). I think it needs to return `null` like before.